prusti-tests/tests/verify/ui/failing-postcondition.stderr

```diff
@@ -1,8 +1,8 @@
 error: [Prusti: verification error] postcondition might not hold.
- --> $DIR/failing-postcondition.rs:8:11
+ --> $DIR/failing-postcondition.rs:8:28
   |
 8 | #[ensures(something_true() && false)]
```

Comment:
What made the span change here?

Reply:
Not sure how that one slipped through! I couldn't find it by just looking so it looks like I'll have to throw in some print statements to track it down.

Reply:
Oh great, looks like [this change](https://github.com/viperproject/prusti-dev/pull/1271/commits/d4069dd6e86805754cfa48090f88e08bac2090e8) messed that up somehow. That doesn't make sense to me because it's already using an explicit span. Any ideas?